lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -22,6 +22,49 @@
 
 LOG_MODULE_REGISTER(lte_lc_helpers, CONFIG_LTE_LINK_CONTROL_LOG_LEVEL);
 
+static int string_to_int(const char *str_buf, int base, int *output)
```

Comment:
Should't this function return an unsigned 32-bit integer if in fact a converted cell_id converts to a number larger than an signed 32-bit integer can hold. Lets say the cell_id is DDD11B07 = 3721468679.

Reply:
Just realized. The maximum E-Cell ID is of course well within a signed 32. So keeping as is should be all good.

Reply:
Yep, this should be fine